plugins/chat/assets/javascripts/discourse/initializers/chat-sidebar.js

```diff
@@ -42,6 +42,59 @@ export default {
     });
 
     withPluginApi("1.3.0", (api) => {
+      api.addSidebarSection(
+        (BaseCustomSidebarSection, BaseCustomSidebarSectionLink) => {
+          const SidebarChatMyThreadsSectionLink = class extends BaseCustomSidebarSectionLink {
+            route = "chat.threads";
+            text = I18n.t("chat.my_threads.title");
+            title = I18n.t("chat.my_threads.title");
+            name = "user-threads";
+            prefixType = "icon";
+            prefixValue = "discourse-threads";
+            suffixType = "icon";
+            suffixCSSClass = "unread";
+
+            constructor() {
+              super(...arguments);
+
+              if (container.isDestroyed) {
+                return;
+              }
+
+              this.chatChannelsManager = container.lookup(
+                "service:chat-channels-manager"
+              );
+            }
+
+            get suffixValue() {
+              return this.chatChannelsManager.publicMessageChannels.some(
+                (channel) => channel.unreadThreadsCount > 0
+              )
+                ? "circle"
+                : "";
+            }
+          };
+
+          const SidebarChatMyThreadsSection = class extends BaseCustomSidebarSection {
+            // we only show `My Threads` link
+            hideSectionHeader = true;
+
+            name = "user-threads";
+
+            // sidebar API doesn’t let you have undefined values
+            // even if you don't show the section’s header
+            title = "";
+
+            get links() {
+              return [new SidebarChatMyThreadsSectionLink()];
+            }
+          };
+
+          return SidebarChatMyThreadsSection;
+        },
+        CHAT_PANEL
+      );
```

Comment:
That's a lot of code to show a link, but couldn't find how to do simpler

Reply:
I had to do the same for admin sidebar too, no way around it.